### PR TITLE
[56] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by Oluwabusiolami**
## Task: `Name of given task`
- `Changes you made to given task`
Capture

### CODE REVIEW

File: submissions/oluwabusolamiEluyela/html-forms/contact.html [hunk 1]
```diff
@@ -5,14 +5,19 @@
     <title>Contact Us!</title>
 </head>
 <body>
-<nav>
+<header>
     <h1>JUSTIFY</h1>
+    <nav>
     <a href="../html/index.html">Product Page</a>
     <a href="../html-forms/Contact.html">Contact Us</a>
```
Comment (Ireti203) [inline submissions/oluwabusolamiEluyela/html-forms/contact.html:12]: I can see Contact.html in your link, but the file name is contact.html. It won't navigate correctly. Kindly correct this. 

Also, you did not add navigation to your Product Page, kindly do that also.

File: submissions/oluwabusolamiEluyela/html-forms/contact.html [hunk 2]
```diff
@@ -24,6 +29,7 @@ <h1>Contact Form</h1>
         <textarea rows="17" cols="50" placeholder="Please type your message here"></textarea><br><br>
         <button>Submit</button>
     </fieldset>
+</section>
 </form>
```
Comment (Ireti203) [inline submissions/oluwabusolamiEluyela/html-forms/contact.html:33]: Why did you use a form closing tag here? I can see an opening tag for form anywhere in your code.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/oluwabusolamiEluyela/html-forms/contact.html [hunk 3]
```diff
@@ -5,14 +5,19 @@
...
+    <nav>
     <a href="../html/index.html">Product Page</a>
     <a href="../html-forms/Contact.html">Contact Us</a>
-</nav>
+    </nav>
+</header>
+<section>
+  <div>
     <h1>Contact Form</h1>
-    <form action="#" method="post" enctype="multipart/form-data">
```
Comment (adaugochi) [inline submissions/oluwabusolamiEluyela/html-forms/contact.html:14]: Is there a reason why you are removing the opening `<form>` tag?

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/oluwabusolamiEluyela/html-forms/contact.html (hunk 2 above)
Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html-forms/contact.html]: Resolved, Please review

File: submissions/oluwabusolamiEluyela/html-forms/contact.html (hunk 3 above)
Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html-forms/contact.html:14]: It was a mistake

File: submissions/oluwabusolamiEluyela/html-forms/contact.html (hunk 1 above)
Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html-forms/contact.html]: Done

File: submissions/oluwabusolamiEluyela/html-forms/contact.html [hunk 4]
```diff
@@ -24,6 +32,6 @@ <h1>Contact Form</h1>
         <textarea rows="17" cols="50" placeholder="Please type your message here"></textarea><br><br>
         <button>Submit</button>
     </fieldset>
-</form>
```
Comment (ToluAkin) [inline submissions/oluwabusolamiEluyela/html-forms/contact.html:27]: @Oluwabusiolami You need to close the form tag

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Oluwabusiolami You need to link the contact page to the product page as wll

**Comment by Oluwabusiolami** [comment]
Changes have been made. Kindly review @adaugochi @ToluAkin @Ireti203

File: submissions/oluwabusolamiEluyela/html-forms/contact.html (hunk 3 above)
Comment (adaugochi) [reply submissions/oluwabusolamiEluyela/html-forms/contact.html:14]: You have two opening form tag

Comment (adaugochi) [reply submissions/oluwabusolamiEluyela/html-forms/contact.html:14]: Kindly go through your code

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Oluwabusiolami** [comment]
> Changes have been made. Kindly review @adaugochi @ToluAkin @Ireti203



> Changes have been made. Kindly review @adaugochi @ToluAkin @Ireti203

@adaugochi I have removed one. 
Kindly highlight all the changes I am to make, so that I can do them once. Thank you.

File: submissions/oluwabusolamiEluyela/html-forms/contact.html (hunk 4 above)
Comment (adaugochi) [inline submissions/oluwabusolamiEluyela/html-forms/contact.html:27]: You need to add a closing form tag

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — CHANGES_REQUESTED** [review]
Kindly update your PR title to the task title that is on the app. Going forward, I will have to close this PR so you can then create a new one

**Pull request closed without merging**